src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -21,6 +21,8 @@
 
     <!--This property sets the default output types supported by the project system.-->
     <SupportedOutputTypes Condition="'$(SupportedOutputTypes)' == ''">Exe;WinExe;Library</SupportedOutputTypes>
+
+    <SuppressOutOfDateMessageOnBuild>true</SuppressOutOfDateMessageOnBuild>
```

Comment:
This was hardcoded into the legacy project system (couldn't override the behavior), but sure, makes sense...